Dockerfile

```diff
@@ -1,4 +1,8 @@
 FROM node:15.5.0
+# Transcrypt dependency
+RUN apt-get update && apt-get install -y bsdmainutils
+# Install transcrypt for EA Forum
+RUN curl -sSLo /usr/local/bin/transcrypt https://raw.githubusercontent.com/elasticdog/transcrypt/v2.1.0/transcrypt && chmod +x /usr/local/bin/transcrypt
```

Comment:
IMO it's good habit, when there's a specific version number anyways, to use a URL that has something cryptographic in it. In this case that would be
https://raw.githubusercontent.com/elasticdog/transcrypt/2f905dce485114fec10fb747443027c0f9119caa/transcrypt which can't be changed even if elasticdog wants to (because it has a commit ID in it, which is a hash of the contents).